src/hotspot/share/cds/archiveBuilder.cpp

```diff
@@ -1294,7 +1294,7 @@ class ArchiveBuilder::CDSMapLogger : AllStatic {
 
       if (source_oop != nullptr) {
         // This is a regular oop that got archived.
-        print_oop_with_requested_addr_cr(&st, source_oop, false);
+        print_oop_with_requested_addr_cr(&st, source_oop, false, true);
```

Comment:
Now that the function takes two boolean args. I think it is good to add comments on those boolean args.

Reply:
I added the comment about what the first `false` parameter means. I also removed the second parameter as suggested by David.